lease/client.go

```diff
@@ -0,0 +1,128 @@
+/*
+   Copyright 2014 CoreOS, Inc.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package lease
```

Comment:
Rather than creating a new top-level package, this code should probably live in `pkg/lease`. It should not be specific to fleet at all, and we should consider moving it to `github.com/coreos/etcd/client` once that package is closer to "done".